android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/requests/ShowConstantTbtTests.java

```diff
@@ -4,7 +4,7 @@
 import com.smartdevicelink.protocol.enums.FunctionID;
```

Comment:
@kostyaBoss I would revert these changes in `ShowConstantTbtTests.java`.  They are valid changes, but are out of scope of the PR.